### ISSUE TITLE
Feature Request: make payload limit configureable 

### ISSUE THREAD

**Issue by udo1toni** [thomasnordquist/MQTT-Explorer#836]
As said in https://github.com/thomasnordquist/MQTT-Explorer/issues/408, it would be nice to have this configurable.

Or (to circumvent memory limitations)
- last message is always complete
- limit historic messages to (e.g.) 20.000 Bytes each and truncate.
- at least mark truncated payloads as truncated, so a user is aware of the situation.

**Comment by hessijames79** [comment]
Yes please. Spent the last 30 minutes to find out if the broker or the client were the culprit of the truncated value just to find out it was MQTT Explorer.

**Comment by ilgrank** [comment]
The author (@thomasnordquist ) has not been seen in quite a long, and even tho for a brief period @bj00rn had joined him to try to resurrect MQTTExplorer, he also seems to have surrendered, alas.

**Comment by wangnick** [comment]
Me too, please. The app is so beautiful, but now one of my (rare) topics is 68k in size ...

**Comment by bj00rn** [comment]
> The author ([@thomasnordquist](https://github.com/thomasnordquist) ) has not been seen in quite a long, and even tho for a brief period [@bj00rn](https://github.com/bj00rn) had joined him to try to resurrect MQTTExplorer, he also seems to have surrendered, alas.

yep life got in the way, im afraid.
